pkg/controller/deployment/sync.go

```diff
@@ -247,14 +240,30 @@ func (dc *DeploymentController) getNewReplicaSet(deployment *extensions.Deployme
 	newRevision := strconv.FormatInt(maxOldRevision+1, 10)
 
 	existingNewRS, err := deploymentutil.FindNewReplicaSet(deployment, rsList)
```

Comment:
Yes, part of a different issue though: https://github.com/kubernetes/kubernetes/issues/31780